sql/core/src/main/scala/org/apache/spark/sql/execution/command/views.scala

```diff
@@ -129,10 +129,10 @@ case class CreateViewCommand(
             sparkSession,
             analyzedPlan,
             aliasedPlan.schema,
-            originalText,
-            child))
+            originalText))
       } else {
-        aliasedPlan
+        assert(isTemporary)
```

Comment:
we are in the branch of `if (viewType == LocalTempView)`, this assert seems not necessary, as the if condition is not far away.

Reply:
Removed.